kolibri/core/analytics/pskolibri/common.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import functools
+import os
+import sys
+from collections import namedtuple
+
+PY3 = sys.version_info[0] == 3
+POSIX = os.name == "posix"
+WINDOWS = os.name == "nt"
+LINUX = sys.platform.startswith("linux")
+MACOS = sys.platform.startswith("darwin")
```

Comment:
I still think all this stuff should be in `kolibri.utils.compat` -- not sure if you've seen my previous comment?

Reply:
I saw it and I answered you previously.
compat was used for Python versions compatibility, not for OS compatibilty.
Also this is a complete new module, used in the analytics part. I don't see any benefit of doing it in compat if it's being used in analytics.

Reply:
Something weird happened with that comment thread, sorry about that. I see it now.

No, never mind moving this.